src/server/pps/server/api_server.go

```diff
@@ -2119,6 +2059,24 @@ func (a *apiServer) CreatePipelineInTransaction(
 				"running, you can run 'pachctl update pipeline --clean' which will " +
 				"delete this open commit")
 		}
+	} else {
+		// Create output and spec repos
+		if err := a.env.PfsServer().CreateRepoInTransaction(txnCtx,
+			&pfs.CreateRepoRequest{
+				Repo:        client.NewRepo(pipelineName),
+				Description: fmt.Sprintf("Output repo for pipeline %s.", request.Pipeline.Name),
+				Update:      true,
```

Comment:
the "Update" in these is identical to what was done before by ignoring AlreadyExists errors, but perhaps the former makes the intent more clear?

Reply:
... except that this clobbers any new description that a user might have set, so I'll revert this